### PR TITLE
Feature/Setup project

### PR DESCRIPTION
**Pull request by zofnow**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,41 @@
+apply plugin: 'com.android.application'
```
Comment (lwasylkowski) [inline app/build.gradle]: Lint rules project doesn't have to be an android application or an android project at all. just `apply plugin: 'kotlin'` is enough. This means you don't need the manifest, android configuration etc.

Unless, of course, you'd like to have two projects here -- lint rules and a test project, if you feel it's helpful. Still lint rules project is pure kotlin

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#Tue Feb 27 12:40:11 CET 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```
Comment (lwasylkowski) [inline gradle/wrapper/gradle-wrapper.properties]: `4.5.1` maybe? 4.6-rc-2 is most recent, but it's rc

File: app/build.gradle (hunk 1 above)
Comment (zofnow) [reply app/build.gradle]: ha right, we're not sure about having the test project here yet, so I'll just remove it for now

Comment (lwasylkowski) [reply app/build.gradle]: Actually the whole module should be called `checks` or `lintChecks` or something like that then as well

**Pull request merged**